doc/api/url.md

```diff
@@ -525,6 +525,45 @@ console.log(JSON.stringify(myURLs));
 // Prints ["https://www.example.com/","https://test.example.org/"]
 ```
 
+#### Static: URL.from(input)
+
+* `input` {Object}
+  * `protocol` {string} any valid protocol. See [`url.protocol`][].
```

Comment:
It seems some links lack bottom references (here and below) and are not rendered as links.

Reply:
Will look them up. By the way how can I render these docs locally?

Reply:
For the quick check, you can use GitHub rendering from the PR files menu "View file", like https://github.com/nodejs/node/blob/68de7d41346f69497f5e853deef999e93ea66834/doc/api/url.md#static-urlfrominput

If you want to make HTML docs as in Node,js site, you can build the docs using `$ NODE=/path/to/node make doc-only` (see https://github.com/nodejs/node/blob/master/BUILDING.md#building-the-documentation or see https://github.com/nodejs/node/issues/19330 if you are on Windows).